mregsite/settings.py

```diff
@@ -46,7 +46,7 @@ def envvar(var: str, default: DefaultT) -> DefaultT:
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = ")e#67040xjxar=zl^y#@#b*zilv2dxtraj582$^(e6!wf++_n#"
 
-LOG_LEVEL = envvar("MREG_LOG_LEVEL", "CRITICAL").upper()
+LOG_LEVEL = envvar("MREG_LOG_LEVEL", "CRITICAL" if TESTING else "ERROR").upper()
```

Comment:
I welcome a better solution to not leak logs from 4xx errors to stderr during tests. Could maybe override the log handler inside the tests instead?